test/unit/worldwide_organisation_test.rb

```diff
@@ -4,11 +4,22 @@
 class WorldwideOrganisationTest < ActiveSupport::TestCase
   include GdsApi::TestHelpers::Worldwide
 
-  context "finding organisations in a location" do
-    should "return organisations for the location" do
-      results = load_fixture('australia')
-      assert results[0].is_a?(WorldwideOrganisation)
-      assert_equal ["UK Trade & Investment Australia", "British High Commission Canberra"], results.map(&:title)
+  context '.for_location' do
+    should 'instantiates WorldwideOrganisation objects using data from the API' do
+      organisations_data = [
+        OpenStruct.new(title: 'organisation-1-title'),
+        OpenStruct.new(title: 'organisation-2-title')
+      ]
+      worldwide_api = stub
```

Comment:
> Minor: I might be inclined to give this stub a name. It would make any error messages easier to understand.

Addressed in https://github.com/alphagov/smart-answers/pull/2471/commits/e4a9921683ee37471df2e9c651a17283471ee99a.